runner/app/live/pipelines/streamdiffusion.py

```diff
@@ -367,8 +367,13 @@ async def _load_image_from_url(url: str) -> Image.Image:
     if not (url.startswith('http://') or url.startswith('https://')):
         raise ValueError(f"Invalid image URL: {url}")
 
+    # Set user-agent to prevent 403 errors from servers like Wikipedia
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (compatible; AI-Runner/1.0; +https://github.com/livepeer/ai-runner)',
```

Comment:
Nice, I like that we're not FULLY pretending we're a browser, if they want they can see from our header that we are not :P

Reply:
haha yeah, actually I checked again and we can just set it to anything at all, just can't be empty. But I guess leaving it like this may help with other slightly more strict servers